standalone/portal-api/src/modules/collateral/models/collaterals.entity.ts

```diff
@@ -12,11 +12,11 @@ export class CollateralEntity extends EntityHelper {
   @Allow()
   facilityType: string;
 
-  @Column({ type: 'decimal', precision: 2, scale: 2, default: 0 })
+  @Column({ type: 'integer' })
```

Comment:
Why is this stored as an integer? The property name (ratio) suggests we're going to store decimal numbers